README.md

```diff
@@ -80,23 +73,13 @@ To cross-build for 2.12 and 2.11, the rewrite rule introduces a dependency on th
 
 ```scala
 // build.sbt
-ThisBuild / scalafixDependencies += "org.scala-lang.modules" %% "scala-collection-migrations" % "<version>"
 libraryDependencies +=  "org.scala-lang.modules" %% "scala-collection-compat" % "<version>"
-addCompilerPlugin(scalafixSemanticdb)
-scalacOptions ++= List("-Yrangepos", "-P:semanticdb:synthetics:on")
```

Comment:
oops, synthetics [are used in the rule](https://github.com/scala/scala-collection-compat/search?q=synthetics), so we'll need to keep that :(